pkg/services/reputation/common/managers/managers.go

```diff
@@ -83,7 +93,7 @@ func (mb *managerBuilder) BuildManagers(epoch uint64, p reputation.PeerID) ([]co
 		}
 	}
 
-	return nil, nil
+	return nil, &errNoSuchPeerInNetmap{peer: p, epoch: epoch}
```

Comment:
What exactly caused this change?

Reply:
a deleted empty managers check in `managers`'s code and moved it here. i think that this is error case when node not in `netmap` but we are building managers for it

Reply:
reverted back